agent/internal/version/version.go

```diff
@@ -8,9 +8,9 @@ import (
 )
 
 var (
-	Version        = "dev"
-	CommitHash     = "n/a"
-	BuildTimestamp = "n/a"
+	Version        = "dev" //nolint:gochecknoglobals
```

Comment:
You add a lot of `//nolint:gochecknoglobals` to the codebase. Can't we add this rule in global go linter config? Could you please check it?

Reply:
The linter treats package private globals as errors too, but in these specific places we need these globals. We can disable the gochecknoglobals linter rule but then it defeats the point :D

Reply:
Okey.